apps/ledger-live-desktop/src/renderer/components/WebPlatformPlayer/index.jsx

```diff
@@ -215,7 +216,7 @@ export default function WebPlatformPlayer({ manifest, onClose, inputs, config }:
           dispatch(
             openModal("MODAL_OPERATION_DETAILS", {
               operationId: optimisticOperation.id,
-              accountId: account.id,
+              accountId: mainAccount.id,
               parentId: null,
```

Comment:
Should not remain as before with the use of parentId parameter?
```js
openModal("MODAL_OPERATION_DETAILS", {
              operationId: optimisticOperation.id,
              accountId: account.id,
              parentId: mainAccount.id,
```

Reply:
True good catch.
The only thing is that I just saw that this modal actually doesn't exist in the codebase.
So I should probably delete this part.

Reply:
I found the change I have to do https://github.com/LedgerHQ/ledger-live-desktop/pull/3844/files#diff-5cbabb2ee78e11940f0209c716c0294035841204349b480ac1c1e7403be85e7f

Reply:
This should be good now